Archive.Zip.UnitTests/ZipFileBuilderTest.cs

```diff
@@ -190,16 +189,23 @@ public void NoHandlerForArchiveError_ThrowsException ()
 
       zipBuilder.AddFile (fileInfoMock);
       var zipFileName = Path.GetTempFileName();
-      try
-      {
-        using (zipBuilder.Build (zipFileName))
-        {
-        }
-      }
-      finally
-      {
-        FileUtility.DeleteAndWaitForCompletion (zipFileName);
-      }
+
+      Assert.That (
+          () =>
+          {
+            try
```

Comment:
The try-finally should probably be outside the assert.that, just the using is inside the Assert.That.